.github/workflows/docker-deploy.yml

```diff
@@ -16,18 +16,32 @@ on:
     - cron: "3 17 * * 4"
 
 jobs:
+  date:
```

Comment:
Maybe we should call this "tag" and not "date", so it's a bit more transparent,what we get from this step. It's main purpose is giving us a nice tag (which might contain a date)

Reply:
fair point, I'll have a look at that